### PR TITLE
feat: design token and theme package documentation

### PR DESCRIPTION
**Pull request by SiTaggart**
Add a page for each of the Design Token package and Theme package, with install instructions and how to use. Similar to a component or utility page.

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: packages/paste-website/src/pages/tokens/design-tokens-package.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+- iOS JSON
+- JavaScript
+  - commonJS
+  - es2015
+- Typescript definitions
+  - commonJS
+  - es2015
+- Sketch palette
+
+A simple way to grab or view the latest files is to visit [https://unpkg.com/browse/@twilio-paste/design-tokens/dist/](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/)
```
Comment (TheSisb) [inline packages/paste-website/src/pages/tokens/design-tokens-package.mdx]: Love this 👍

File: packages/paste-website/src/components/shortcodes/component-header/index.tsx [hunk 2]
```diff
@@ -7,16 +7,27 @@ import {SidebarCategoryRoutes} from '../../../constants';
 import {P} from '../../Typography';
 import {Heading} from '../../Heading';
 
+const getCategoryNameFromRoute = (categoryRoute: string): string => {
```
Comment (TheSisb) [inline packages/paste-website/src/components/shortcodes/component-header/index.tsx]: Nice

File: packages/paste-website/src/pages/tokens/design-tokens-package.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
+---
+title: Design Tokens Package
+package: '@twilio-paste/design-tokens'
+description: The Paste design tokens package allows you to install Paste design tokens in a variety of different formats
```
Comment (aayushpi) [inline packages/paste-website/src/pages/tokens/design-tokens-package.mdx]: Where is this description showing up?

Comment (aayushpi) [reply packages/paste-website/src/pages/tokens/design-tokens-package.mdx]: Nit, remote "Paste", sentence is too redundant.

File: packages/paste-website/src/pages/tokens/design-tokens-package.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  data={props.data.allPasteDesignTokensPackage.edges}
+/>
+
+***
+
+<content>
+
+The Paste design tokens are available as an NPM package to download and install the latest design tokens and values from a central source of truth.
+
+Paste currently exports design tokens into different formats to be used in a variety of technology stacks. The formats we currently support are:
```
Comment (aayushpi) [inline packages/paste-website/src/pages/tokens/design-tokens-package.mdx]: This can be added to the previous paragraph, this page reads very fragmented.

File: packages/paste-website/src/pages/tokens/design-tokens-package.mdx (hunk 1 above)
Comment (aayushpi) [inline packages/paste-website/src/pages/tokens/design-tokens-package.mdx]: I would add this to the top paragraph so that folks can explore links quickly.

File: packages/paste-website/src/pages/tokens/design-tokens-package.mdx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+- Typescript definitions
+  - commonJS
+  - es2015
+- Sketch palette
+
+A simple way to grab or view the latest files is to visit [https://unpkg.com/browse/@twilio-paste/design-tokens/dist/](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/)
+
+## Themes
+
+Paste supports multiple themes and as such exports the same token formats for each theme. Each file in the root of the design tokens package is re exported for each theme with the corresponding values updated for each token.
```
Comment (aayushpi) [inline packages/paste-website/src/pages/tokens/design-tokens-package.mdx]: Simplify:

Paste supports multiple themes. This is done by exporting the same token formats for each theme. Each file in the root of the design tokens package is exported for each theme with the corresponding values updated for each token.

For example:
`Add example`

File: packages/paste-website/src/pages/tokens/design-tokens-package.mdx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+You can access the tokens for a given theme by installing the design tokens package and importing the file from the theme folder that you want to use.
+
+You can view the themes we currently support in design tokens at [https://unpkg.com/browse/@twilio-paste/design-tokens/dist/themes/](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/themes/).
```
Comment (aayushpi) [inline packages/paste-website/src/pages/tokens/design-tokens-package.mdx]: What does "in design tokens" mean? Do we support themes in any other way? If not, would rewrite to "You can view the themes we currently support at:"

Comment (aayushpi) [reply packages/paste-website/src/pages/tokens/design-tokens-package.mdx]: Also, it feels weird that the themes we support are linked in a place external to the doc site. Can't we just add here?

File: packages/paste-website/src/pages/tokens/theme-package.mdx [hunk 7]
```diff
@@ -0,0 +1,122 @@
+---
+title: Theme Package
+package: '@twilio-paste/theme'
+description: The Paste theme package allows easy and simple access to the paste design tokens via a theme provider to react components
```
Comment (aayushpi) [inline packages/paste-website/src/pages/tokens/theme-package.mdx]: nit: capitalize `React`

Comment (aayushpi) [reply packages/paste-website/src/pages/tokens/theme-package.mdx]: nit: Capitalize `Theme Package` maybe?

File: packages/paste-website/src/pages/tokens/theme-package.mdx [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+  data={props.data.allPasteThemePackage.edges}
+/>
+
+***
+
+<content>
+
+## Theme Provider
+
+Paste Components require you to wrap your application in the Paste Theme provider to apply the theme tokens to the components you use, and provide a way to access Paste Design Tokens in your custom components.
```
Comment (aayushpi) [inline packages/paste-website/src/pages/tokens/theme-package.mdx]: Break this into two sentences, it's a little hard to read.

File: packages/paste-website/src/pages/tokens/theme-package.mdx [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+
+### Using tokens in custom components
+
+Sometimes you will need to create something custom that is not available in Paste but you need that component to still look like it's from Paste. The use of Paste Tokens in your styles is the way that you can access global design properties used in all Paste Components.
+
+#### Emotion Styled Component
+
+By using the `Theme.Provider`, when you create a custom component using Emotion CSS, the theme object is available on `props` via context. You can access those via the [Styled-System `themeGet` utility](https://styled-system.com/api#themeget) or props object directly.
+
+This is the preferred method.
```
Comment (aayushpi) [inline packages/paste-website/src/pages/tokens/theme-package.mdx]: Add this to the previous paragraph. Reads like a fragment

File: packages/paste-website/src/pages/tokens/theme-package.mdx [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+
+const custom = styled.div`
+  background-color: ${themeGet('backgrounds.primary')};
+  padding: ${props => props.theme.spacing.spacing20};
+`
+```
+
+#### Paste ThemeConsumer
+
+Paste Theme provides a React Context Consumer for the theme called `Theme.Consumer`. The `Theme.Consumer` takes a function as a child, which provides the theme object as an argument.
```
Comment (aayushpi) [inline packages/paste-website/src/pages/tokens/theme-package.mdx]: Add a line about when would a customer use this as opposed to preferred method.

File: packages/paste-website/src/pages/tokens/theme-package.mdx [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+
+const HookExampleComponent = (): React.ReactElement => {
+  const theme = useTheme();
+  return <p>What is the default text color {theme.textColors.colorText}</p>;
+};
+```
+
+#### Higher Order Component
+
+Paste also provides a HOC Component, which is a wrapper around the Emotion [HOC Component](https://emotion.sh/docs/emotion-theming#withthemecomponent-reactcomponenttype-reactcomponenttype) to be able to access the theme object.
```
Comment (aayushpi) [inline packages/paste-website/src/pages/tokens/theme-package.mdx]: Add a line about when would a customer use this as opposed to preferred method.

File: packages/paste-website/src/pages/tokens/theme-package.mdx [hunk 12]
```diff
@@ -0,0 +1,122 @@
...
+<Theme.Consumer>
+  {({theme}) => {
+    return <p>What is the default text color {theme.textColors.colorText}</p>;
+  }}
+</Theme.Consumer>
+```
+
+#### Paste useTheme Hook
+
+Paste Theme provides a React Hook called `useTheme` which returns the theme object from the React Context via the Theme Provider.
```
Comment (aayushpi) [inline packages/paste-website/src/pages/tokens/theme-package.mdx]: Add a line about when would a customer use this as opposed to preferred method.

File: packages/paste-website/src/pages/tokens/design-tokens-package.mdx (hunk 3 above)
Comment (SiTaggart) [reply packages/paste-website/src/pages/tokens/design-tokens-package.mdx]: > Where is this description showing up?

Meta description, so Google and when you paste the url into Slack.

File: packages/paste-website/src/pages/tokens/design-tokens-package.mdx (hunk 6 above)
Comment (SiTaggart) [reply packages/paste-website/src/pages/tokens/design-tokens-package.mdx]: > Do we support themes in any other way?

Yes. We export tokens in formats, so for example iOS. For iOS there is an iOS based file for each theme that we support. (console: https://unpkg.com/browse/@twilio-paste/design-tokens@2.4.0/dist/tokens.ios.json , sendgrid: https://unpkg.com/browse/@twilio-paste/design-tokens@2.4.0/dist/themes/sendgrid/tokens.ios.json)

We also provide a React Theme specifically for use with react applications. That is the `theme` package.

This site doesn't _host_ anything, it's just documentation. So whjy is it weird that we link to a compiled hosted asset?

File: packages/paste-website/src/pages/tokens/theme-package.mdx (hunk 10 above)
Comment (SiTaggart) [reply packages/paste-website/src/pages/tokens/theme-package.mdx]: Gunna bundle the "reason" for all three under one umbrella, because it's if you're not using emotion or for whatever reason you want the token value in some logic.

File: packages/paste-website/src/pages/tokens/design-tokens-package.mdx (hunk 6 above)
Comment (aayushpi) [reply packages/paste-website/src/pages/tokens/design-tokens-package.mdx]: This makes more sense. I would reword to say:

Tokens are supported for different formats:
- iOS [Link]
- Android [Link]
- Nokia 3310 [Link]
- etc.

Within Themes, we support them presently for:
- Console [Link]
- SendGrid [Link]

File: packages/paste-website/src/pages/tokens/theme-package.mdx (hunk 10 above)
Comment (aayushpi) [reply packages/paste-website/src/pages/tokens/theme-package.mdx]: Great!

File: packages/paste-website/src/pages/tokens/theme-package.mdx [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+
+const custom = styled.div`
+  background-color: ${themeGet('backgrounds.primary')};
+  padding: ${props => props.theme.spacing.spacing20};
+`
+```
+
+### Alternatives when not using Emotion
+
+If you are not using Emotion styled components or want to access the values of tokens in your React components outside of styling, there are 3 ways this can be achieved.
```
Comment (aayushpi) [inline packages/paste-website/src/pages/tokens/theme-package.mdx]: nit: swap 3 with three. We follow AP styleguide editorially (which Kyle's Tone of Voice will cover), so all numbers till and including nine will be spelled in editorial context. 10 and above will be numerical.

**Comment by SiTaggart** [comment]
@aayushpi if you want to give it a pass I've done a once through now

File: packages/paste-website/src/pages/tokens/theme-package.mdx [hunk 14]
```diff
@@ -0,0 +1,146 @@
...
+
+const ExampleComponent = ({theme}) => (
+  <p>What is the default text color {theme.textColors.colorText}</p>
+);
+const ExampleComponentwithTheme = withTheme(ExampleComponent);
+```
+
+***
+
+<Box marginTop={30}>
```
Comment (TheSisb) [inline packages/paste-website/src/pages/tokens/theme-package.mdx]: Does this use the token or is it setting 30px? 😕

Comment (richbachman) [reply packages/paste-website/src/pages/tokens/theme-package.mdx]: Error on my part when I added the changelogs. It should be `space90`.

File: packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx [hunk 15]
```diff
@@ -0,0 +1,137 @@
...
+
+## Design Primitive Components
+
+Design Primitive Components are React components that expose style properties using [Styled-System](https://www.styled-system.com). These style properties are typed React props that only expose design tokens as values. There are style properties for each of the categories that are included on the [theme object](theme-package#theme) from the [theme package](theme-package).
+
+We use these components internally to create almost everything in Paste. When used, they provide an incredible number of different combinations of design tokens that can be used to create new, unique things that will look exactly like they came directly from Paste. They also provide support for the themes that Paste offers out of the box and a free upgrade path to any new design language updates in the future. Your custom components will never be left behind.
+
+Each Design Primitive is an Emotion Styled Component that creates styles that are encapsulated to component. This way you can style your custom components with the confidence that your styles should never clash with another teams work.
```
Comment (richbachman) [inline packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx]: Should we link to Emotion here?

File: packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx [hunk 16]
```diff
@@ -111,7 +112,26 @@ const SidebarNavigation: React.FC<SidebarNavigationProps> = () => {
...
+            Design Tokens
+            <SiteNavAnchorArrow isOpen={tokensOpen} />
+          </SiteNavButton>
+          <SiteNavNestList isOpen={tokensOpen}>
+            <SiteNavItem>
+              <SiteNavAnchor to={SidebarCategoryRoutes.TOKENS}>Token list</SiteNavAnchor>
+            </SiteNavItem>
+            <SiteNavItem>
+              <SiteNavAnchor to={`${SidebarCategoryRoutes.TOKENS}/how-to-compose-custom-ui-with-tokens`}>
+                How to compose custom UI with tokens
```
Comment (TheSisb) [inline packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx]: image
This is kind of long.  Can we rename it to `Composing with tokens`?

File: packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx [hunk 17]
```diff
@@ -0,0 +1,137 @@
+---
+title: How to compose custom UI with tokens
+description: You may encounter times where you are required to build an experience that isn't covered by out of the box Paste components. This guide aims to help you create custom experiences whilst still using the design system.
```
Comment (TheSisb) [inline packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx]: whilst -> while? 🤕

File: packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx [hunk 18]
```diff
@@ -0,0 +1,137 @@
...
+
+<p>{props.pageContext.frontmatter.description}</p>
+
+</content>
+
+***
+
+<content>
+
+Paste will never aim to cover every experience that Twilio ships to customers, it will make our very small but mighty team an incredible bottle neck to you doing your job and we don't want that. We also fully expect that there will be certain, very product specific experiences that would make little sense as generic components or design patterns that can be used in any product vertical. With that in mind we have hopefully devised some ways that will help you and your team ship your products whilst still being able to use assets from the design system.
```
Comment (TheSisb) [inline packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx]: whilst -> while? 🥇 
If we want to use whilst, I think this reads better as `ship your products whilst using assets from the design system`

File: packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx [hunk 19]
```diff
@@ -0,0 +1,137 @@
...
+
+## The happy path
+
+Obviously the first option is to work with your design partner and identify all the existing Paste components that can be used to create the experience. If you are struggling or have quesitons come to Design System Office Hours or post in the #help-design-system slack channel, we'll get you back on track.
```
Comment (TheSisb) [inline packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx]: `quesitons` -> `questions`

Comment (TheSisb) [reply packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx]: Should we link to info on Office Hours?

File: packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx [hunk 20]
```diff
@@ -0,0 +1,137 @@
...
+  >
+    Some child elements
+  </Box>
+</Box>`}
+</LivePreview>
+
+
+### Text
+
+The text component is designed to be a general purpose text utilty. It too exposes the `as` prop to allow it to be transformed into any valid HTML element and comes with a number of text styling properties. These properties include font-size, text color, weight and line height and allow you to create text styles that should match the design system.
```
Comment (TheSisb) [inline packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx]: > and line height and allow

Suggestion: remove the second "and" and make that a new sentence.

File: packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx [hunk 21]
```diff
@@ -0,0 +1,137 @@
...
+
+<LivePreview scope={{Box, Text}} language="jsx">
+  {`<Box padding="space20">
+  <Text as="p" marginBottom="space40">Some text</Text>
+  <Text as="h1" fontSize="fontSize70" fontWeight="fontWeightMedium" lineHeight="lineHeight70" mb="space80">Custom heading</Text>
+  <Text as="ul" ml="space60">
```
Comment (TheSisb) [inline packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx]: Should this be a Text or a Box?

If we're making it a Text, can we pass the fontSize to it and would the children elements receive the styling?

File: packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx [hunk 22]
```diff
@@ -0,0 +1,137 @@
...
+    <Text as="li" fontSize="fontSize30">Custom list item</Text>
+    <Text as="li" fontSize="fontSize30">Custom list item</Text>
+  </Text>
+  <Text as="p" textColor="colorTextError" fontSize="fontSize20" fontWeight="fontWeightBold">Custom Error Message</Text>
+</Box>`}
+</LivePreview>
+
+### Compositions
+
+You can use combinations of Box and Text to create custom compositions, utilizing the design token values from Paste. This should mean that your custom components should look like that are part of the same design language as the rest of the application. These automatically respond to the theme that the provider has set, try the theme switcher in the site header to see how the examples below respond to switching the theme.
```
Comment (TheSisb) [inline packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx]: `compositions, utilizing`
No comma necessary

File: packages/paste-website/src/pages/tokens/design-tokens-package.mdx [hunk 23]
```diff
@@ -0,0 +1,74 @@
...
+  categoryRoute={SidebarCategoryRoutes.TOKENS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/button"
+  data={props.data.allPasteDesignTokensPackage.edges}
+/>
+
+***
+
+<content>
+
+The Paste design tokens are available as an NPM package to download and install the latest design tokens and values from a central source of truth. A simple way to grab or view the latest files is to visit [https://unpkg.com/browse/@twilio-paste/design-tokens/dist/](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/). Tokens are supported for different formats within the NPM package:
```
Comment (TheSisb) [inline packages/paste-website/src/pages/tokens/design-tokens-package.mdx]: This opening sentence reads weird.
How about just:
> The Paste design tokens are available as an NPM package as the central source of truth.

File: packages/paste-website/src/pages/tokens/design-tokens-package.mdx [hunk 24]
```diff
@@ -0,0 +1,74 @@
...
+  - [es2015](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/tokens.es6.d.ts)
+- [Sketch palette](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/sketch.sketchpalette)
+
+## Themes
+
+Paste supports multiple themes. This is done by exporting the same token formats for each theme. Each file in the root of the design tokens package is exported for each theme with the corresponding values updated for each token. For example:
+
+| Default theme                   | SendGrid theme                  |
+|---------------------------------|---------------------------------|
+| $color-brand-highlight: #F22F46 | $color-brand-highlight: #3368FA |
```
Comment (TheSisb) [inline packages/paste-website/src/pages/tokens/design-tokens-package.mdx]: Would be cool to put a second example here for a non-color token, like spacing and/or font for the full effect

File: packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx [hunk 25]
```diff
@@ -111,7 +112,26 @@ const SidebarNavigation: React.FC<SidebarNavigationProps> = () => {
...
+              </SiteNavAnchor>
+            </SiteNavItem>
+            <SiteNavItem>
+              <SiteNavAnchor to={`${SidebarCategoryRoutes.TOKENS}/design-tokens-package`}>Tokens package</SiteNavAnchor>
```
Comment (TheSisb) [inline packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx]: > Tokens package" 

Should we call this "Design Tokens Package" to match how we write design-tokens elsewhere?

File: packages/paste-website/src/pages/tokens/theme-package.mdx [hunk 26]
```diff
@@ -0,0 +1,146 @@
...
+  data={props.data.allPasteThemePackage.edges}
+/>
+
+***
+
+<content>
+
+## Theme
+
+Paste components are built using [Emotion](https://emotion.sh/) and [Styled System](https://styled-system.com) and providing a Theme object containing our design tokens. This enables our components to be themed, and also provide styling properties to our some of our design primtive components like `Box` and `Text`. The theme object is grouped by categories and for each category, each key represents a camelCase formatted design token name from the [tokens list](/tokens/).
```
Comment (TheSisb) [inline packages/paste-website/src/pages/tokens/theme-package.mdx]: " and providing " -> "and provide"

Comment (TheSisb) [inline packages/paste-website/src/pages/tokens/theme-package.mdx]: "primtive" -> "primitive"

File: packages/paste-website/src/pages/tokens/theme-package.mdx [hunk 27]
```diff
@@ -0,0 +1,146 @@
...
+
+For examle the `$color-background-success` token would be available on the `backgrounds` key as `colorBackgroundSuccess`.
```
Comment (TheSisb) [inline packages/paste-website/src/pages/tokens/theme-package.mdx]: "examle" -> "example"

File: packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx (hunk 20 above)
Comment (TheSisb) [inline packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx]: "utilty" -> "utility"

File: packages/paste-website/src/pages/tokens/theme-package.mdx (hunk 27 above)
Comment (TheSisb) [inline packages/paste-website/src/pages/tokens/theme-package.mdx]: This sentence is confusing.  Where is `backgrounds` used in the following example?  Isn't it the `backgroundColors` key?

File: packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx (hunk 19 above)
Comment (SiTaggart) [reply packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx]: > Should we link to info on Office Hours?

Do we have a link?

File: packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx (hunk 21 above)
Comment (SiTaggart) [reply packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx]: So `Text` right now has a set default font-size, which matches what the system classes as the default body size. The reason being is that we font set a body size as you can't really override someone's application, and we don't actually know where this will be used so can't rely on any inheritance.

So even if I did set it on the ul, it won't currently affect the children as they set their own font-size anyway.

File: packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx (hunk 15 above)
Comment (SiTaggart) [reply packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx]: Sure!

File: packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx (hunk 16 above)
Comment (SiTaggart) [reply packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx]: Seems like an improvement to me but I'll wait for a consensus before changing it as it involves changing filenames and stuff and there maybe other ideas. @aayushpi @serifluous @two24studios for input

File: packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx (hunk 25 above)
Comment (SiTaggart) [reply packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx]: probably

File: packages/paste-website/src/pages/tokens/theme-package.mdx (hunk 27 above)
Comment (SiTaggart) [reply packages/paste-website/src/pages/tokens/theme-package.mdx]: Yup

File: packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx (hunk 19 above)
Comment (TheSisb) [reply packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx]: https://wiki.hq.twilio.com/display/PD/Paste%3A+A+Design+System+for+Twilio#Paste:ADesignSystemforTwilio-WeeklyOfficeHours

File: packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx (hunk 21 above)
Comment (TheSisb) [reply packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx]: Makes sense.  Is there any guidelines or issues with making a `ul` element a `Text` as opposed to a `Box` though?

**Comment by aayushpi** [comment]
One down, three to go - design tokens package is mostly edited

**Comment by aayushpi** [comment]
Added:
* Text changes to Theme Package
* Text changes to Design Tokens Package
* Small rewrite to Guide for Custom UI
* Rewrote package descriptions for theme and token.
